docs/reference/errors-and-warnings/NU1605.md

```diff
@@ -12,10 +12,35 @@ f1_keywords:
 
 # NuGet Warning NU1605
 
+## Example 1
+
 <pre>Detected package downgrade: 'PackageB' from 4.0.0 to 3.5.0. Reference the package directly from the project to select a different version.<br/>  'PackageA' 3.5.0 -> 'PackageB' 3.5.0<br/>  'PackageC' 4.0.0 -> 'PackageD' 4.0.0 -> 'PackageB' 4.0.0</pre>
 
 ### Issue
-A dependency package specified a version constraint on a higher version of a package than restore ultimately resolved. That is, because of the "nearest wins" rule when resolving packages, a nearer package in the graph may have overridden a distant package.
+A dependency package specified a version constraint on a higher version of a package than restore ultimately resolved. That is because of the [nearest-wins](https://docs.microsoft.com/en-us/nuget/consume-packages/dependency-resolution#nearest-wins) rule - when resolving packages, the version of the nearer package in the graph will override that of the distant package with the same ID.
+
+### Solution
+To the project exhibiting the restore error, add a package reference to the higher version of the package.
+
+In the example above, you would add a package reference to Package B version 4.0.0:
+
+<pre>
+  'PackageA' 3.5.0 -> 'PackageB' 3.5.0<br/>
+  'PackageC' 4.0.0 -> 'PackageD' 4.0.0 -> 'PackageB' 4.0.0<br/>
+  'PackageB' 4.0.0
+</pre>
+
+Since, [nearest-wins](https://docs.microsoft.com/en-us/nuget/consume-packages/dependency-resolution#nearest-wins), the direct package reference to PackageB v4.0.0, will take precedence over the transitive reference to PackageB v3.5.0.
+
+## Example 2
+
+<pre>Detected package downgrade: Microsoft.NETCore.App from 2.1.8 to 2.1.0. Reference the package directly from the project to select a different version.<br/>
+  test -> mvc -> Microsoft.NETCore.App (>= 2.1.8)<br/>
+  test -> Microsoft.NETCore.App (>= 2.1.0)<br/>
+</pre>
+
+### Issue
+The mvc project specified a version constraint on a higher version of a package than restore ultimately resolved. That is because of the [nearest-wins](https://docs.microsoft.com/en-us/nuget/consume-packages/dependency-resolution#nearest-wins) rule - when resolving packages, the version of the nearer package in the graph will override that of the distant package with the same ID.
 
 ### Solution
-Add a direct reference to the project for the higher version of the package that you want to use.
\ No newline at end of file
+This specific error (with Microsoft.NETCore.App package) is improved my moving your .NET Core SDK to 2.2.100 or later. Microsoft.NETCore.App is an auto-referenced package that the .NET Core SDK before version 3.0.100 chooses to bring in automatically. Also see (Self-contained deployment runtime roll forward)[https://docs.microsoft.com/en-us/dotnet/core/deploying/runtime-patch-selection].
```

Comment:
https://review.docs.microsoft.com/en-us/help/contribute/links-how-to?branch=master#links-to-articles-in-other-repos-or-docsets

Reply:
also note that the link here has the syntax backwards -- it should be `[text](http://link)`, not `(text)[http://link]`